lang/en.txt

```diff
@@ -6040,5 +6040,10 @@ Cast#
 4086 View item details#
 4087 Hide item details#
 4088 You found %d gold pieces in the chest!#
+# Nav Prompts
+4089 Move cursor#
+4093 View Spells#
+4094 View Items#
+4095 View Skills#
```

Comment:
I used a couple of these for crab-related messages. Just a heads up that these will have to change when we merge

Reply:
Moved language[4089] to [4092], crab messages 4089,4090 should be fine now, just need to combine these files at merge time